### PR TITLE
Added pagination for local JSON and removed axios dummy URl calls

### PR DESCRIPTION
**Pull request by ManikandanS910**

### CODE REVIEW

File: autozone-poc/src/components/Shops/product-list/ProductList.js [hunk 1]
```diff
@@ -17,16 +16,10 @@ class ProductList extends Component {
     this.handlePageClick = this.handlePageClick.bind(this);
     }
 
-    receivedData() {
-        axios
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Shops/product-list/ProductList.js:21]: Please make as reuseable code. Try to implement as possibility usecase

File: autozone-poc/src/components/Shops/product-list/ProductList.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import React, { Component, Fragment } from 'react';
-import axios from 'axios'
 import ReactPaginate from 'react-paginate';
 import './ProductList.css';
 import Product from './Product.js';
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Shops/product-list/ProductList.js:4]: It is possible to convert into functional component. Also, add props validation in the page.

File: autozone-poc/src/components/Shops/product-list/ProductList.js [hunk 3]
```diff
@@ -75,29 +67,4 @@ class ProductList extends Component {
     }
 }
 
-export default ProductList;
-
-// import React, { Fragment } from 'react';
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Shops/product-list/ProductList.js:80]: Remove this unused code

**Review by gokulwebdeveloper — CHANGES_REQUESTED** [review]

File: autozone-poc/src/components/Shops/product-list/ProductList.js (hunk 3 above)
Comment (ManikandanS910) [reply autozone-poc/src/components/Shops/product-list/ProductList.js:80]: Already removed code

File: autozone-poc/src/components/Shops/product-list/ProductList.js (hunk 2 above)
Comment (ManikandanS910) [reply autozone-poc/src/components/Shops/product-list/ProductList.js:4]: Changed to functional component and added props validation

File: autozone-poc/src/components/Shops/product-list/ProductList.js (hunk 1 above)
Comment (ManikandanS910) [reply autozone-poc/src/components/Shops/product-list/ProductList.js:21]: commented on removed code

**Review by gokulwebdeveloper — APPROVED** [review]
Please remove commented line while you integrating redux

**Pull request merged**